sdk/digitaltwins/Azure.DigitalTwins.Core/tests/E2eTestBase.cs

```diff
@@ -38,7 +39,7 @@ protected DigitalTwinsClient GetClient(DigitalTwinsClientOptions options = null)
         {
             if (options == null)
             {
-                options = new DigitalTwinsClientOptions();
+                options = new DigitalTwinsClientOptions(){ Retry = { Delay = TimeSpan.Zero, Mode = RetryMode.Fixed}};
```

Comment:
InstrumentClientOptions?

Reply:
it's actually happening a few lines below